Spec/Auth.swift

```diff
@@ -4449,16 +4415,17 @@ override class var defaultTestSuite : XCTestSuite {
                         }
                     }
                 }
-            }
             
             // RSA4f, RSA8c
-            context("when the JWT token is returned with application/jwt content type") {
+            
 
-                beforeEach {
+                func beforeEach__JWT_and_rest__when_the_JWT_token_is_returned_with_application_jwt_content_type() {
```

Comment:
Here's an example of how the migrator handles `beforeEach` hooks. It turns each call into a function, and then invokes this function at the start of all of the affected `test__…` methods.

(It does the same kind of thing for `afterEach`, too.)

TODO link to more complicated example.

Reply:
I'm curious why not https://developer.apple.com/documentation/xctest/xctest/1500341-setup ? 
>Before each test begins, XCTest calls setUp(completion:) for asynchronous state preparation, then setUpWithError(), followed by this method. Override this method to reset state for each test method.

Reply:
Good question. The reason I didn't use `setUp` / `tearDown` is that, since Quick allows you to have `context`-level `before/afterEach` hooks, each test case might have its own setup and teardown steps, depending on which `context`s it's nested within. (See the link in the grandparent comment for an example of this.)

Now, that doesn't fully answer your question, I realise. We _could_ turn the _file-wide_ `before/afterEach` hooks (e.g. those which are contained within `spec`) into `setUp` / `tearDown` methods, and then in the test cases we'd only have to call any `context`-specific hooks. But I don't think we can get away from having to call _some_ hooks directly in the test cases. Would you like me to look into making this change?

Reply:
This is something that could be changed later. Leave it as it is. Thanks for the explanation.